File: stress/t/flt_newton.f
      PROGRAM FNEWTON
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I,N
      TARGET=612.0D0
      G=25.0D0
      N=20
      DO 10 I=0,N-1
        T=TARGET/G
        G=0.5D0*(G+T)
   10 CONTINUE
      WRITE(*,'(A,F0.6)') 'g= ',G
      WRITE(*,'(A,F0.6)') 'ref= ',SQRT(TARGET)
      END

File: stress/t/flt_newton.c
#include <stdio.h>
#include <math.h>
int main(void){
  long i,n=20; double target=612.0,g=25.0,t;
  for(i=0;i<n;i++){ t=target/g; g=0.5*(g+t); }
  printf("g= %f\n",g);
  printf("ref= %f\n",sqrt(target));
  return 0;
}
